### PR TITLE
Allow you to configure the flush period

### PR DESCRIPTION
**Pull request by kevinburke**

### CODE REVIEW

File: src/librato.coffee [hunk 1]
```diff
@@ -12,7 +12,10 @@ librato.configure = (newConfig) ->
   config = newConfig
   collector = new Collector()
   client = new Client config
-  worker = new Worker job: librato.flush
+  if newConfig.period
```
Comment (bobzoller) [inline src/librato.coffee]: you can just always pass `period` and remove this `if` block -- src/worker.coffee:8 handles not defined just fine.

**Comment by bobzoller** [comment]
small style nit

thanks for this and especially for including a test.

**Comment by kevinburke** [comment]
Ahh alright, should be updated now (I rebased back down to one commit)

**Comment by bobzoller** [comment]
merged, thanks again.  travis should publish v2.1.0 shortly.

**Pull request merged**